gui/src-tauri/Cargo.toml

```
@@ -39,10 +39,10 @@ tempfile = "3"
 which = "4.3"
 glob = "0.3"
 open = "4"
-shadow-rs = { version = "0.22", default-features = false }
+shadow-rs = { version = "0.23", default-features = false }
 const_format = "0.2.30"
 cfg-if = "1"
-anyhow = "1.0"
+color-eyre = { version = "0.6", default-features = false }
```

```suggestion
color-eyre = "0.6"
```
I would use the defaults